### PR TITLE
feat(react-devtools): update to react-devtools-inline 

### PR DESCRIPTION
**Pull request by danilowoz**
CodeSandbox logoCodeSandbox logo&nbsp; Open in CodeSandbox <a href="https://codesandbox.io/p/github/codesandbox/sandpack/draft/competent-vaughan">Web Editor</a> | <a href="https://codesandbox.io/p/vscode?owner=codesandbox&repo=sandpack&branch=draft/competent-vaughan">VS Code</a>

As Sandpack recently supports React@18, we're ready now to support the latest version of react-devtools-inline, in order to be used on React Docs.

### CODE REVIEW

**Comment by danilowoz** [comment]
Hey @bvaughn, by any chance, do you know why I'm now getting the following error? Does it look familiar to you? I remember facing this issue before, but when we implemented the latest version in another branch, it was fixed. However, migrating the previous implementation to the main branch, I'm facing this again.

Screenshot 2022-07-20 at 21 05 11

**Comment by bvaughn** [comment]
Looks like the wrong version of React is being imported? Needs to be one of the _experimental_ releases b'c of the unstable APIs DevTools is still using.

File: sandpack-react/package.json [hunk 1]
```diff
@@ -54,17 +54,17 @@
     "codesandbox-import-util-types": "^2.2.3",
     "lodash.isequal": "^4.5.0",
     "lz-string": "^1.4.4",
-    "react-devtools-inline": "4.4.0",
+    "react-devtools-inline": "4.25.0",
```
Comment (bvaughn) [inline sandpack-react/package.json:57]: I'm guessing the version bump added a dependency on an unreleased/unstable React API.

**Comment by danilowoz** [comment]
Ohh I thought React 18 would support all the react-devtools-inline dependencies. So definitely, this is the issue because I'm using 18 as the main version. Thanks!

**Comment by bvaughn** [comment]
Not yet! React DevTools is still using some _suspense_ APIs that aren't finalized. Also `useTransition` and `useDeferredValue` which I think aren't stable yet either? I don't remember what's in stable. I've lived on the bleeding edge for too long.

**Pull request closed without merging**